### PR TITLE
[AC][SUPP-316] Audit BARS check

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

**Review by barnesjake — DISMISSED** [review]

File: test/testsupport/stubs/BarsStub.scala [hunk 1]
```diff
@@ -30,7 +30,7 @@ object BarsStub {
 
     def accountNumberNotWellFormatted(): StubMapping = stubOk(validateUrl, ValidateJson.accountNumberNotWellFormatted)
 
-    def sortCodeNotPresentOnEiscd(): StubMapping = stubOk(validateUrl, ValidateJson.sortCodeNotPresentOnEiscd)
+    def sortCodeNotPresentOnEiscd(): StubMapping = stubOk(validateUrl, ValidateJson.accountNumberNotWellFormatted)
```
Comment (darrell-plant) [inline test/testsupport/stubs/BarsStub.scala:33]: Was this a mistake?

File: test/controllers/BankDetailsControllerSpec.scala [hunk 2]
```diff
@@ -399,16 +460,18 @@ class BankDetailsControllerSpec extends ItSpec {
...
             BarsStub.ValidateStub.accountNumberNotWellFormatted()
-            List(("Enter a valid combination of bank account number and sort code", "#bars"))
+            List(("Enter a valid combination of bank account number and sort code", "#bars")) ->
+              ValidateJson.accountNumberNotWellFormatted
 
           case "sortCodeNotPresentOnEiscd" =>
             BarsStub.ValidateStub.sortCodeNotPresentOnEiscd()
-            List(("Enter a valid combination of bank account number and sort code", "#bars"))
+            List(("Enter a valid combination of bank account number and sort code", "#bars")) ->
+              ValidateJson.accountNumberNotWellFormatted
```
Comment (darrell-plant) [inline test/controllers/BankDetailsControllerSpec.scala:474]: and here. Something's not right I think

File: test/testsupport/stubs/BarsStub.scala (hunk 1 above)
Comment (AndyHWChung) [reply test/testsupport/stubs/BarsStub.scala:33]: oh yeah, whoops I'll revert it back

File: test/controllers/BankDetailsControllerSpec.scala (hunk 2 above)
Comment (AndyHWChung) [reply test/controllers/BankDetailsControllerSpec.scala]: corrected [here](https://github.com/hmrc/essttp-frontend/pull/107/commits/175f2805bac302d37abbab5bb5c0f2684eb2904c)

File: test/testsupport/stubs/BarsStub.scala (hunk 1 above)
Comment (AndyHWChung) [reply test/testsupport/stubs/BarsStub.scala]: not sure how that happened - corrected now [here](https://github.com/hmrc/essttp-frontend/pull/107/commits/175f2805bac302d37abbab5bb5c0f2684eb2904c)

**Review by darrell-plant — APPROVED** [review]

**Pull request merged**